src/allmydata/web/map-update-status.xhtml

```diff
@@ -1,35 +1,38 @@
-<html xmlns:n="http://nevow.com/ns/nevow/0.1">
+<html xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1">
   <head>
     <title>Tahoe-LAFS - Mutable File Servermap Update Status</title>
     <link href="/tahoe.css" rel="stylesheet" type="text/css"/>
     <link href="/icon.png" rel="shortcut icon" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   </head>
+
   <body>
 
-<h1>Mutable File Servermap Update Status</h1>
+    <h1>Mutable File Servermap Update Status</h1>
+
+    <ul>
+      <li>Started: <span t:render="started"/></li>
+      <li>Finished: <span t:render="finished"/></li>
+      <li>Storage Index: <span t:render="si"/></li>
+      <li>Helper?: <span t:render="helper"/></li>
+      <li>Progress: <span t:render="progress"/></li>
+      <li>Status: <span t:render="status"/></li>
+    </ul>
 
-<ul>
-  <li>Started: <span n:render="started"/></li>
-  <li>Finished: <span n:render="finished"/></li>
-  <li>Storage Index: <span n:render="si"/></li>
-  <li>Helper?: <span n:render="helper"/></li>
-  <li>Progress: <span n:render="progress"/></li>
-  <li>Status: <span n:render="status"/></li>
-</ul>
+    <h2>Update Results</h2>
 
-<h2>Update Results</h2>
-<ul>
-  <li n:render="problems" />
-  <li>Total: <span n:render="time" n:data="time_total" /></li>
-  <ul>
-    <li>Initial Queries: <span n:render="time" n:data="time_initial_queries" /></li>
-    <li n:render="privkey_from" />
-    <li>Cumulative Verify: <span n:render="time" n:data="time_cumulative_verify" /></li>
-  </ul>
-  <li n:render="server_timings" />
-</ul>
+    <ul>
+      <t:transparent t:render="problems" />
```

Comment:
might be more-clear if this (and the other t:transparent) were just `<li>` as before (and can return `tag(...)` in the renderers instead of `tag(li(..))`

Reply:
I tried that, and it rendered an empty bullet.  There must be a better way?

Reply:
Ahhh, okay .. didn't notice that one of the cases is "show nothing" .. leave the `transparent` then :) sorry for the noise